src/tools/kata-ctl/src/ops/volume_ops.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2022 Boston University
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::args::{DirectVolSubcommand, DirectVolumeCommand};
+
+use anyhow::{anyhow, Ok, Result};
+use futures::executor;
+use hyper;
+use kata_types::mount::DirectVolumeMountInfo;
+use std::{fs, path::Path, path::PathBuf, time::Duration};
+use url;
+
+use runtimes;
+
+const TIMEOUT: Duration = Duration::new(5, 0);
+const CONTENT_TYPE_JSON: &str = "application/json";
+const MOUNT_INFO_FILE_NAME: &str = "mountInfo.json";
```

Comment:
This file should reference `mountInfoFileName` defined in `src/runtime/pkg/direct-volume/utils.go` rather than defining its own version.

Reply:
Could you give some hint on how to do this?🤔 I'm thinking that one way to do this is to move `mountInfoFileName` and `kataDirectVolumeRootPath` from `src/runtime/pkg/direct-volume/utils.go` to some shared makefile and do code generation upon compilation. 
Or maybe I should just update the katactl's makefile to fetch the actual value of these variables each time katactl is compiled? But this would break the moment the original go code gets changed.

Reply:
Hi @tzY15368 - just re-read my comment and realised it certainly could be perceived as confusing! :smile: Sorry about that. What I meant was that if the runtime is generating that file, we should use the same definition as the runtime. But if the rust runtime (runtime-rs) isn't yet generating that file (as #5369 is still open) I guess we should put those definitions in `src/lib/kata*` somewhere. But wdyt @Tim-0731-Hzt ?

My worry is that if we create the const values in the kata-ctl tool, they might get re-added for #5369, and _those_ versions might end up changing at some point in the future leading to a disconnect (aka bugs ;)

Reply:
how about put them in `src/libs/kata_types/src/mount.rs` ?

Reply:
Thanks for the clarification! 
> how about put them in `src/libs/kata_types/src/mount.rs` ?
Putting them in `kata-types` does make a lot of sense, will do right away 😀